src/rules/noSubmoduleImportsRule.ts

```diff
@@ -102,10 +99,6 @@ class NoSubmoduleImportsWalker extends Lint.AbstractWalker<string[]> {
     }
 }
 
-function isAbsoluteOrRelativePath(path: string): boolean {
-    return /^(..?(\/|$)|\/)/.test(path);
```

Comment:
This one was actually wrong. it needed `\.` instead of just `.`
Anyways, we now use a typescript function which also check for `C:\foo\bar` as of typescript@2.5.0